app/routes/search/documents.js

```diff
@@ -0,0 +1,167 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { isEmpty } from '@ember/utils';
+import { action } from '@ember/object';
+import { warn } from '@ember/debug';
+import { startOfDay, endOfDay, parse } from 'date-fns';
+import search from 'frontend-kaleidos/utils/mu-search';
+import Snapshot from 'frontend-kaleidos/utils/snapshot';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
+export default class CasesSearchRoute extends Route {
```

Comment:
`DocumentsSearchRoute`

Reply:
Renaming to `SearchDocumentsRoute` because Ember generates class names like that based on the route